data/korath/korath variants.txt

```diff
@@ -993,6 +993,7 @@ ship "Ra'at Ik 621" "Ra'at Ik 621 (Storm)"
 	turret "Firestorm Battery"
 
 
+
```

Comment:
I think the best practise here would be to consider the different bay arrangements of RAI 621s variants of the same ship rather than different ships.

```suggestion
```